Architectural Styles and Patterns/Homework Task 1.txt

```diff
@@ -0,0 +1,3 @@
+The architectural style suitable for this solution is a Service-Oriented Architecture (SOA). Here's the explanation for this architecture style choice:
```

Comment:
one of the biggest concept of the SOA is to have the single enterprise service bus to communication between services, but I don't see it on the diagram you have provided, would you mind adding it there?

Reply:
This text describe illustrated  architecture on homework task

Reply:
discussed, resolved